FeedStoreChallenge/CoreDataModels/CoreDataFeedImage.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  CoreDataFeedImage.swift
+//  FeedStoreChallenge
+//
+//  Created by Guilherme Bueno on 16/05/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import CoreData
+
+@objc(CoreDataFeedImage)
+final class CoreDataFeedImage: NSManagedObject {
+	@NSManaged var id: UUID
+	@NSManaged var imageDescription: String?
+	@NSManaged var location: String?
+	@NSManaged var url: URL
+	@NSManaged var feed: CoreDataFeed
```

Comment:
`feed` is optional in `FeedStore.xcdatamodel`, please make it optional.

```suggestion
	@NSManaged var feed: CoreDataFeed?
```